scripts/fhir-works-local-develop.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
```

Comment:
I tried using `yalc` links to do local development, but this is a very tedious process. Every time you make a change to a local package:
1. You have to do `yalc push` to push it to the local package manager. 
2. You have to remove all node modules in `development` package. 
3. You then relink the dependent packages in the `deployment` package. 
4. You then do yarn install.

Using `yarn` links is much faster.